pkgs/top-level/python-packages.nix

```diff
@@ -4009,6 +4011,8 @@ in {
 
   hpack = callPackage ../development/python-modules/hpack { };
 
+  hpccm = callPackage ../development/python-modules/hpccm { };
```

Comment:
If this is an application, not a python library please remove this entry.

Reply:
Hello Sandro, this is both a python application and package.